cost-analyzer/values.yaml

```
@@ -360,6 +360,24 @@ pricingCsv:
     URI: s3://kc-csv-test/pricing_schema.csv  # a valid file URI
     csvAccessCredentials: pricing-schema-access-secret
 
+# Enterprise custom pricing, enterprise key required
+enterpriseCustomPricing:
+  # Enabled the feature by setting this flag to "true". If enabled, the feature
+  # requires a CSV pricing spec to be provided at the location, below, using a
+  # ConfigMap, for which we provide instructions, also below.
+  enabled: false
+  # location:
+  #  URI: /var/configs/enterprise-pricing/spec.csv  # local ConfigMap
```

As a best practice, avoid commenting out configurations. It's preferred to explicitly define them, even if they're disabled. In the example below, setting `enabled: false` ensures the related configs won't be templated:

```suggestion
  location:
    URI: /var/configs/enterprise-pricing/spec.csv  # local ConfigMap
```

Here are additional style guidelines I wrote up. Let me know if you disagree though! https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/DEVELOPMENT.md#yaml-style-guidelines